cfgov/cfgov/settings/base.py

```diff
@@ -465,14 +465,6 @@
 LOGIN_REDIRECT_URL = '/login/welcome/'
 LOGIN_URL = "/login/"
 
-
```

Comment:
We still need this code.

Reply:
Where does this code still get used?

Reply:
for the rate checker; that flag hasn't been flipped yet.

Reply:
Right, that path still [uses](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/urls.py#L86) `SheerTemplateView`, but that doesn't seem to depend on the Sheer Site at all. All that's really doing is modifying how/where the template file gets loaded from.

Reply:
The flag has now been flipped in production.